actuator/src/main/java/org/tron/core/utils/ProposalUtil.java

```diff
@@ -328,6 +339,28 @@ public static void validator(DynamicPropertiesStore dynamicPropertiesStore,
         }
         break;
       }
+      case ALLOW_TVM_STAKE: {
+          if (!forkController.pass(ForkBlockVersionEnum.VERSION_4_1)) {
+          throw new ContractValidateException(
+              "Bad chain parameter id [ALLOW_TVM_STAKE]");
+        }
+        if (value != 1 && value != 0) {
+          throw new ContractValidateException(
+              "This value[ALLOW_TVM_STAKE] is only allowed to be 1 or 0");
+        }
+        break;
+      }
+      case ALLOW_TVM_ASSET_ISSUE: {
```

Comment:
There is another proposal called `ALLOW_TVM_TRANSFER_TRC10`.
Should this be called `ALLOW_TVM_TRC10_ISSUE` in proposal scope.